src/backend/processor/metadata.ts

```diff
@@ -11,14 +11,23 @@ import {
     workspace
 } from 'vscode';
 import * as path from 'path';
-import { parseMetadata } from '../parser';
-import { CheckerMetadata } from '../types';
+import { parseCheckerData, parseMetadata } from '../parser';
+import { CheckerData, CheckerMetadata } from '../types';
+import { ExtensionApi } from '../api';
 
 export class MetadataApi implements Disposable {
     private _metadata?: CheckerMetadata;
     public get metadata(): CheckerMetadata | undefined {
         return this._metadata;
     }
+    private _checkerData: Map<string, CheckerData> = new Map();
+    /**
+     * Content based on the CodeChecker checkers command.
+     * Key: [analyzer name, checker name], value: checker descriptor data (incl. doc URL)
+     */
+    public get checkerData(): Map<string, CheckerData>{
```

Comment:
Is the type "string" correct in the Map the light of the above comment: "Key: [analyzer name, checker name],...."?

Reply:
Whoops that was a leftover, corrected.